src/App.tsx

```diff
@@ -1,16 +1,28 @@
+import { useState } from 'react';
+
 import './App.scss';
+
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
+
 import moviesFromServer from './api/movies.json';
 
-export const App = () => {
+import { Movie } from './types/Movie';
+
+export const App: React.FC = () => {
+  const [visibleMovies, setVisibleMovies] = useState(moviesFromServer);
+  const addNewMovie = (movie: Movie): void => {
```

Comment:
it's unnecessary, TS already knows about it
```suggestion
  const [visibleMovies, setVisibleMovies] = useState(moviesFromServer);
  
  const handleAdd = (movie: Movie) => {
```